.gitignore

```diff
@@ -2,6 +2,21 @@
 node_modules
 tmp
 
+# ignore js, map, and css in element dirs
+elements/*/*.js
+elements/*/*.map
+elements/*/*.css
+# but don't ignore tooling files
+!elements/*/gulpfile.js
+!elements/*/rollup.config.js
+
+# and same for themes
+themes/*/*.js
```

Comment:
@mwcz Can we set the entire themes repo to ignore for now since it's being deprecated anyway?

Reply:
I think that should happen in a dedicated PR, one dedicated to deprecating `themes`.